curriculum/challenges/english/07-scientific-computing-with-python/learn-list-comprehension-by-building-a-case-converter-program/657effaa2a5e0277d71f9cbe.md

```diff
@@ -7,13 +7,15 @@ dashedName: step-8
 
 # --description--
 
-By the end of the loop, `snake_cased_char_list` should contain all the converted characters in correct order. Use the `.join()` string method to convert the list of characters into a string.
+By this point, the `snake_cased_char_list` variable contains all the converted characters. To convert this list of characters into a string, you'll use the `.join()` method. For example:
```

Comment:
Just a suggestion to reorder the instruction a bit.

```
By this point, the variable `snake_cased_char_list` holds the list of converted characters. To combine these characters into a single string, you should utilize the `join()` method. 

The `join` method works by concatenating each element of the list into a string, separated by a designated string, known as the separator. 

```py
# join all characters using empty string
result_string = ''.join(characters)
```

Now, right after the `for` loop, use the `join()` method to join the elements in `snake_cased_char_list` using an empty string as the separator. Assign the result to a new variable named `snake_cased_string`.

```